### ISSUE TITLE
Access to fetch at 'https://api.mainnet-beta.solana.com/' from origin 'https://nftwebsite.com' has been blocked

### ISSUE THREAD

**Issue by YrgkenKoutsi** [ilmoi/nft-armory#14]
Hi guys,

Has anyone encountered this issue when deploying the app in production

```
Access to fetch at 'https://api.mainnet-beta.solana.com/' from origin 'https://nftwebsite.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: It does not have HTTP ok status
```

**Comment by ilmoi** [comment]
Yeah Solana changed something in their networking settings, the mainnet node doesn't work from prod front ends anymore. You'll have to ask them on discord or use a custom node.

**Comment by YrgkenKoutsi** [comment]
> Yeah Solana changed something in their networking settings, the mainnet node doesn't work from prod front ends anymore. You'll have to ask them on discord or use a custom node.

Thank you for the reply I appreciate it! 

One more question and probably a silly one but I was just wondering as I noticed you can mint as many copies as you want as long as you got an Arweave / IPFS URI though on solscan it will show max supply 1 on each image even if you got 10 the same ones but how can you tell which one is the original one and is there a way to prevent minting multiple ones from the same uri?
